gpu_reliability/platforms/gcp.py

```diff
@@ -40,6 +40,7 @@ def __init__(
         credentials = Credentials.from_service_account_info(loads(service_account))
         self.image_client = compute_v1.ImagesClient(credentials=credentials)
         self.instance_client = compute_v1.InstancesClient(credentials=credentials)
+        self.zone_operations_client = compute_v1.ZoneOperationsClient(credentials=credentials)
```

Comment:
cc @akramer - doing this during the class initialization, because we need to authorize with the same credential variables. From my tests it seems like this still works in the wait function, but let me know if there's some reason why you opted to initialize inline versus putting it in the `__init__` originally.

Reply:
It was purely lack of familiarity with the code. I tested on a GCE instance using the local service account, and the python API grabs credentials automatically. This fix LGTM and is what I should have done in the first place.